### PR TITLE
Java doc sticky header for the left frame

### PR DESCRIPTION
**Pull request by lavenal**
#### What was fixed?  (Issue # or description of fix)
Issue #317 
Issue #324 
#### Were the changes tested on
- [x] Firefox (Desktop)
- [x] Safari (Desktop)
- [x] Chrome (Desktop)
- [x] Internet Explorer (Desktop)
- [ ] iOS (Mobile)
- [ ] Android (Mobile)
#### Running validation tools
- [ ] https://validator.w3.org/checklink
- [ ] https://validator.w3.org
- [ ] Dymanic Accessability Plugin (DAP)

### CODE REVIEW

File: src/main/content/_assets/js/javadocs.js [hunk 1]
```diff
@@ -306,6 +359,9 @@ function addClickListener(contents) {
             window.history.pushState(state, null, hashParams);
         }
     })
+    contents.find("a").on("contextmenu", function(event) {
+        console.log("event", event);
+    })
```
Comment (steven1046) [inline src/main/content/_assets/js/javadocs.js]: This was meant for a different issue right?

File: src/main/content/_assets/js/javadocs.js [hunk 2]
```diff
@@ -58,6 +58,9 @@ function addExpandAndCollapseToggleButtons() {
         if(isTopLeftPackageIFrame && $(this).contents().find(".toggle").length === 0) {
             var list = $(this).contents().find('ul[title="Packages"]');
             var header = $(this).contents().find("h2[title='Packages']");
+            // header.css("position", "fixed");
+            // header.css("top", "0");
+            // header.css("width", "100%");
```
Comment (steven1046) [inline src/main/content/_assets/js/javadocs.js]: Can you please remove the commented out code?

File: src/main/content/_assets/js/javadocs.js [hunk 3]
```diff
@@ -122,6 +125,9 @@ function addExpandAndCollapseToggleButtonForPackageFrame(contents, leftBottom) {
     // for string comparison.
     var header2_text = header2.text().replace('/\s/g',' ').trim();
     if(header2_text === "All Classes") {
+        // header2.css("position", "fixed");
+        // header2.css("top", "0");
+        // header2.css("width", "100%");
```
Comment (steven1046) [inline src/main/content/_assets/js/javadocs.js]: Can you please remove the commented out code?

File: src/main/content/_assets/js/javadocs.js (hunk 1 above)
Comment (lavenal) [reply src/main/content/_assets/js/javadocs.js]: oops, I still have the codes there to investigate the right click. Thanks for catching it.

File: src/main/content/_assets/js/javadocs.js (hunk 2 above)
Comment (lavenal) [reply src/main/content/_assets/js/javadocs.js]: Sure

File: src/main/content/_assets/js/javadocs.js (hunk 3 above)
Comment (lavenal) [reply src/main/content/_assets/js/javadocs.js]: Sure

**Comment by lavenal** [comment]
Assign Steven to review the latest changes.

**Pull request merged**